modules/button/react/lib/IconButton.tsx

```diff
@@ -81,7 +81,9 @@ const IconButton: ButtonOrAnchorComponent<
     width: size === 'small' ? spacing.l : spacing.xl,
     height: size === 'small' ? spacing.l : spacing.xl,
     borderRadius:
-      variant === IconButtonVariant.Square || variant === IconButtonVariant.SquareFilled
+      variant === IconButtonVariant.Square ||
+      variant === IconButtonVariant.SquareFilled ||
+      variant === IconButtonVariant.ToolbarSquare
```

Comment:
This is starting to get big enough to warrant a separate function for readability (IMO).

Something like:
```js
function getBorderRadius(variant) {
  switch (variant) {
    case IconButtonVariant.Square:
    case IconButtonVariant.SquareFilled:
    case IconButtonVariant.ToolbarSquare:
      return borderRadius.m;
    default:
      return borderRadius.circle;
  }
}
```

Reply:
ok i really like this approach because now that function can be edited in the future if need be, thanks for the suggestion!